ansible/roles/ocp4-workload-mlops/defaults/main.yml

```diff
@@ -0,0 +1,109 @@
+---
+become_override: no
+ocp_username: opentlc-mgr
+silent: False
+
+num_users: 10
+
+module_titles:
+  - { name: m1, title: "Click here to get started", path: /workshop/mlops/lab/kubernetes-overview }
+
+guid: 1002
```

Comment:
There should not be a default value for `guid`. This should simply be required.

Reply:
Updated